### PR TITLE
CI: use apt instead of dnf

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
because it's an ubuntu based image instead of a fedora one

### CODE REVIEW

**Comment by sdroege** [comment]
How does anything work at all?

**Comment by GuillaumeGomez** [comment]
I guess it doesn't.

**Comment by bilelmoussaoui** [comment]
> How does anything work at all?

which things? the docs CI? it doesn't after my PR. The thing is this job step is only triggered on master/on release that's why

**Comment by sdroege** [comment]
Oh I see

File: .github/workflows/docs.yml
Comment (sdroege) [inline .github/workflows/docs.yml:44]: Nowadays you'd usually just use `apt` instead of `apt-get` FWIW

**Review by sdroege — APPROVED** [review]

File: .github/workflows/docs.yml
Comment (bilelmoussaoui) [reply .github/workflows/docs.yml:44]: Actually, let's move the wget install to the docker image, there's no reason to re-install it everytime here

**Comment by bilelmoussaoui** [comment]
don't merge until #140 is merged and the image got updated

**Comment by bilelmoussaoui** [comment]
@GuillaumeGomez this can be merged now as well :)

**Review by GuillaumeGomez — APPROVED** [review]

**Pull request merged**
